plugins/inputs/redis_sentinel/README.md

```diff
@@ -1,7 +1,13 @@
 # Redis Sentinel Input Plugin
 
-A plugin for Redis Sentinel to monitor multiple Sentinel instances that are
-monitoring multiple Redis servers and replicas.
+This plugin collects metrics for [Redis Sentinel][sentinel] instances monitoring
+Redis servers and replicas.
+
+⭐ Telegraf v1.22.0
+🏷️ server
+💻 all
+
+[sentinel]: https://redis.io/docs/latest/operate/oss_and_stack/management/sentinel/
```

Comment:
Linter is failing on this line - 'long line in paragraph'

Reply:
Ignore those for now! The hand-crafted linter is not up to the point. I will switch to markdownlint for detecting long lines which is much better in distinguishing when a long-line limitation makes sense.